SAM7s_base/usart_at91/usart_device_at91.c

```diff
@@ -142,9 +134,17 @@ static unsigned int createUartMode(unsigned int bits, unsigned int parity, unsig
 int usart_device_init()
 {
     AT91F_PMC_EnablePeriphClock( AT91C_BASE_PMC,(1 << AT91C_ID_US0) | (1 << AT91C_ID_US1));
-    if (!initQueues()) return 0;
+
+    /* Init device 0 (Telemetry) */
+    if (!init_txrx_queue(0, 512, 512))
+            return 0;
     usart_device_init_0(8, 0, 1, TELEMETRY_BAUDRATE);
+
+    /* Init device 1 (GPS) */
+    if (!init_txrx_queue(1, 512, 128))
```

Comment:
Rx queue should be larger than Tx queue.  Tx Queue can be minimal, like 16. MK1 does not communicate with GSP module, only receives data.

suggest: init_txrx_queue(1, 16, 512)

Also, extract magic numbers to #defines